openapi_test.go

```diff
@@ -460,25 +460,25 @@ func BenchmarkServer_generateOpenAPI(b *testing.B) {
 }
 
 func TestValidateJsonSpecURL(t *testing.T) {
-	require.Equal(t, true, validateSpecURL("/path/to/jsonSpec.json"))
-	require.Equal(t, true, validateSpecURL("/spec.json"))
-	require.Equal(t, true, validateSpecURL("/path_/jsonSpec.json"))
-	require.Equal(t, false, validateSpecURL("path/to/jsonSpec.json"))
-	require.Equal(t, false, validateSpecURL("/path/to/jsonSpec"))
-	require.Equal(t, false, validateSpecURL("/path/to/jsonSpec.jsn"))
+	require.True(t, validateSpecURL("/path/to/jsonSpec.json"))
+	require.True(t, validateSpecURL("/spec.json"))
+	require.True(t, validateSpecURL("/path_/jsonSpec.json"))
+	require.False(t, validateSpecURL("path/to/jsonSpec.json"))
+	require.False(t, validateSpecURL("/path/to/jsonSpec"))
+	require.False(t, validateSpecURL("/path/to/jsonSpec.jsn"))
 }
 
 func TestValidateSwaggerUrl(t *testing.T) {
-	require.Equal(t, true, validateSwaggerURL("/path/to/jsonSpec"))
-	require.Equal(t, true, validateSwaggerURL("/swagger"))
-	require.Equal(t, true, validateSwaggerURL("/Super-useful_swagger-2000"))
-	require.Equal(t, true, validateSwaggerURL("/Super-useful_swagger-"))
-	require.Equal(t, true, validateSwaggerURL("/Super-useful_swagger__"))
-	require.Equal(t, true, validateSwaggerURL("/Super-useful_swaggeR"))
-	require.Equal(t, false, validateSwaggerURL("/spec.json"))
-	require.Equal(t, false, validateSwaggerURL("/path_/swagger.json"))
-	require.Equal(t, false, validateSwaggerURL("path/to/jsonSpec."))
-	require.Equal(t, false, validateSwaggerURL("path/to/jsonSpec%"))
+	require.True(t, validateSwaggerURL("/path/to/jsonSpec"))
+	require.True(t, validateSwaggerURL("/swagger"))
+	require.True(t, validateSwaggerURL("/Super-useful_swagger-2000"))
+	require.True(t, validateSwaggerURL("/Super-useful_swagger-"))
+	require.True(t, validateSwaggerURL("/Super-useful_swagger__"))
+	require.True(t, validateSwaggerURL("/Super-useful_swaggeR"))
+	require.False(t, validateSwaggerURL("/spec.json"))
+	require.False(t, validateSwaggerURL("/path_/swagger.json"))
+	require.False(t, validateSwaggerURL("path/to/jsonSpec."))
+	require.False(t, validateSwaggerURL("path/to/jsonSpec%"))
```

Comment:
```suggestion
	assert.True(t, validateSwaggerURL("/path/to/jsonSpec"))
	assert.True(t, validateSwaggerURL("/swagger"))
	assert.True(t, validateSwaggerURL("/Super-useful_swagger-2000"))
	assert.True(t, validateSwaggerURL("/Super-useful_swagger-"))
	assert.True(t, validateSwaggerURL("/Super-useful_swagger__"))
	assert.True(t, validateSwaggerURL("/Super-useful_swaggeR"))
	assert.False(t, validateSwaggerURL("/spec.json"))
	assert.False(t, validateSwaggerURL("/path_/swagger.json"))
	assert.False(t, validateSwaggerURL("path/to/jsonSpec."))
	assert.False(t, validateSwaggerURL("path/to/jsonSpec%"))
```

Reply:
Why?

Reply:
Oh I see, as they are parallel assertion that don't rely on the previous ones, this is better to assert all and have the report for each one instead of panicking on the first fail.